packages/amplify-cli-core/API.md

```diff
@@ -888,9 +959,27 @@ export const getPackageManager: (rootPath?: string) => PackageManager | null;
 // @public (undocumented)
 export const getPermissionsBoundaryArn: (env?: string) => string | undefined;
 
+// @public (undocumented)
+export function getPlugin(context: $TSContext, pluginName: string): string | undefined;
+
+// @public (undocumented)
+export function getPluginInstance(context: $TSContext, pluginName: string): any;
+
+// @public (undocumented)
+export function getProjectConfig(): any;
+
+// @public (undocumented)
+export const getProjectDetails: () => IAmplifyProjectDetails;
+
+// @public (undocumented)
+export const getProjectMeta: () => $TSMeta;
+
 // @public (undocumented)
 export function green(message: string): void;
 
+// @public (undocumented)
+export const handleValidGraphQLAuthError: (context: $TSContext, message: string) => Promise<boolean>;
```

Comment:
does this belong to cli-core ?

Reply:
https://github.com/aws-amplify/amplify-cli/pull/12069#discussion_r1117375855